frontend/index.html

```diff
@@ -531,15 +527,56 @@
         // and without this change, even if we set the dateVal to ''
         // we would try to load ntrips_purpose__default.png
         const imgFile = "plots/" + metric + "_" + dateVal + program + ".png";
+        const htmlFile = "plots/" + metric + "_" + dateVal + program + ".html";
         const altTextFile = "plots/" + metric + "_" + dateVal + program + ".txt";
```

Comment:
They do have the same information, maybe if we implement the html as swapping in for the chart instead of being it's own grid element then we can move forward with it being a 1:1 swap for the alt text?

Reply:
As per the meeting discussion, we are keeping both HTML as well as alt text, while *not* modifying any the changes to showcase toggling between chart and html table.